src/CommonUtils/Sirius/Logging/Logger.php

```diff
@@ -2,23 +2,10 @@
 
 namespace CommonUtils\Sirius\Logging;
 
-use Zend\Db\TableGateway\Exception\RuntimeException;
```

Comment:
If the code was using a RuntimeException could it be pre-fixed with a \?

Reply:
`RuntimeException` isn't being used in this class. Safe to remove.